addon/services/store.js

```diff
@@ -0,0 +1,18 @@
+import require, { has } from 'require';
+import StoreMixin from '../mixins.store';
+
+let Store;
+let ExportedStore;
+
+const HAS_EMBER_DATA_PACKAGE = has('ember-data');
+
+if (!HAS_EMBER_DATA_PACKAGE) {
+  Store = require('@ember-data/store').default;
+  ExportedStore = Store.extend(StoreMixin);
+} else {
+  // the re-open in the initializer will handle configuration
+  // in this case
+  ExportedStore = require('ember-data/store').default;
```

Comment:
This won't work until we have our own store service.  I don't mind landing a WIP PR here but only if the unsupported case fails fast (ie a user build should fail if they lack the ember data package until we actually have a store service that supports that case w/o the mixin)